contracts/staking/ValidatorManager.sol

```diff
@@ -122,7 +217,7 @@ abstract contract ValidatorManager is
     function _initializeValidatorRegistration(
```

Comment:
Can you add a comment here explaining why this is the only validator registration/delisting function that needs the `initializedValidatorSet` modifier? I know you explained why in comments in those functions, but I think it would be helpful to have that more visible too.